README.md

```diff
@@ -49,8 +49,8 @@ RxKeyboard.instance.frame
 
     ```swift
     RxKeyboard.instance.visibleHeight
-      .drive(onNext: { keyboardVisibleHeight in
-        scrollView.contentInset.bottom = keyboardVisibleHeight
+      .drive(onNext: { [weak self] keyboardVisibleHeight in
```

Comment:
```suggestion
      .drive(onNext: { [weak scrollView] keyboardVisibleHeight in
```

Reply:
Is it ok to not weakify the scrollView here ?
```swift
.drive(onNext: { [scrollView] keyboardVisibleHeight in
```